README.md

```diff
@@ -1,120 +1,93 @@
+</a>
+</p>
+</br>
 
-See the [installation instructions](https://docs.microsoft.com/windows/apps/windows-app-sdk/set-up-your-development-environment), and guidelines on [creating your first WinUI 3 app](https://docs.microsoft.com/windows/apps/winui/winui3/create-your-first-winui3-app). 
+## 📺 WinUI Community Call
+The WinUI Community Call is your opportunity to learn about WinUI and to engage with the WinUI team and community. Join us online on YouTube at the [Windows Developer channel](https://www.youtube.com/playlist?list=PLI_J2v67C23ZqsolUDaHoFkF1GKvGrttB).
 
-The source code for WinUI 3 can be found [here](https://github.com/microsoft/microsoft-ui-xaml/tree/winui3/release/1.4-stable).
+</br>
 
-## Using WinUI 2
-You can download and use WinUI packages in your app using the NuGet package manager: see the [Getting Started with the Windows UI Library](https://docs.microsoft.com/uwp/toolkits/winui/getting-started) page for more information.
+## 📢 Contributing to WinUI
 
-The source code for WinUI 2 can be found [here](https://github.com/microsoft/microsoft-ui-xaml/tree/winui2/main).
+The WinUI team welcomes your feedback! To understand how we handle incoming feature requests and bugs, please see our [contribution handling](docs/contribution_handling.md) guidelines.
 
-### Packages
+For information on how to contribute, please see [Contributing to WinUI](CONTRIBUTING.md).
 
-| NuGet Package | Build Status | Latest Versions | Documentation |
-| --- | --- | --- | --- |
-| [Microsoft.UI.Xaml](https://www.nuget.org/packages/Microsoft.UI.Xaml) <br /> Controls and Fluent Design for UWP apps | [![Build Status](https://dev.azure.com/ms/microsoft-ui-xaml/_apis/build/status/WinUI-Public-MUX-CI?branchName=main)](https://dev.azure.com/ms/microsoft-ui-xaml/_build/latest?definitionId=20?branchName=main) | [![latest stable version](https://img.shields.io/nuget/v/Microsoft.UI.Xaml.svg)](https://www.nuget.org/packages/Microsoft.UI.Xaml) <br /> [![latest prerelease version](https://img.shields.io/nuget/vpre/Microsoft.UI.Xaml.svg)](https://www.nuget.org/packages/Microsoft.UI.Xaml/absoluteLatest) | [2.8 release](https://docs.microsoft.com/windows/apps/winui/winui2/release-notes/winui-2.8) |
-| [Microsoft.UI.Xaml.Core.Direct](https://www.nuget.org/packages/Microsoft.UI.Xaml.Core.Direct) <br /> Low-level APIs for middleware components | | [![latest prerelease version](https://img.shields.io/nuget/vpre/Microsoft.UI.Xaml.Core.Direct.svg)](https://www.nuget.org/packages/Microsoft.UI.Xaml.Core.Direct/absoluteLatest) | [2.0 prerelease](https://docs.microsoft.com/uwp/api/microsoft.ui.xaml.core.direct) |
+</br>
 
-You can also build a WinUI package yourself from source. See [Contributing to the Windows UI Library](CONTRIBUTING.md) for more information on building and contributing to WinUI.
+## 🛣️ Roadmap
 
-## Documentation
+For info on the WinUI release schedule and high level plans please see the [WinUI Roadmap](https://aka.ms/winappsdk/plans).
```

Comment:
Shouldn't this link to the roadmap document?

Reply:
@eduardobragaxz  Can you point me to that?

Reply:
The aka.ms/winappsdk/plans link now points to the Roadmap. (We meant to update that sooner.) So I think sticking with this link should be good.